homeassistant/components/myuplink/sensor.py

```diff
@@ -0,0 +1,100 @@
+"""Sensor for myUplink."""
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from myuplink.models import DevicePoint
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfTemperature
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import MyUplinkDataCoordinator
+from .const import DOMAIN
+from .coordinator import CoordinatorData
+from .entity import MyUplinkEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class MyUplinkDeviceSensorEntityDescription(SensorEntityDescription):
+    """Describes MyUplink device sensor entity."""
+
+    value_fn: Callable[[CoordinatorData, str], StateType]
+
+
+DEVICE_POINT_DESCRIPTIONS = {
+    "°C": SensorEntityDescription(
+        key="data_point",
```

Comment:
```suggestion
        key="celsius",
```

Reply:
This commit was lost when rebasing.

There's no need to rebase every time you push. Just rebase on latest dev if you know that there are things we need or if it's been a long time since last push.